docs/integrations/sources/sparkpost.md

```diff
@@ -0,0 +1,44 @@
+# SparkPost
+The SparkPost connector for Airbyte enables seamless integration with SparkPost’s email delivery service, allowing users to automatically sync email performance data, including delivery, templates, and click metrics etc into their data warehouses.
+
+## Authentication
+Before you can use SparkPost's REST API you will need to have a valid SparkPost API key. Follow [these](https://support.sparkpost.com/docs/getting-started/create-api-keys) steps to create a SparkPost API key.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `events` | `string` | Events.  |  |
+| `recipients` | `string` | Recipients.  |  |
+| `templates` | `string` | Templates.  |  |
+| `domain` | `string` | Domain.  |  |
+| `from` | `string` | From.  |  |
+| `metrics` | `array` | Metrics.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| search_message_events | event_id | No pagination | ✅ |  ❌  |
+| sending_domains | domain | No pagination | ✅ |  ❌  |
+| ab_test | id | No pagination | ✅ |  ❌  |
+| templates | id | No pagination | ✅ |  ❌  |
+| recipients | id | No pagination | ✅ |  ❌  |
+| metrics_summary |  | No pagination | ✅ |  ❌  |
+| subaccounts | id | DefaultPaginator | ✅ |  ❌  |
+| engagement_details |  | No pagination | ✅ |  ❌  |
+| rejection_reason_metrics |  | DefaultPaginator | ✅ |  ❌  |
+| metrics |  | No pagination | ✅ |  ❌  |
+| metrics_by_template | template_id | DefaultPaginator | ✅ |  ❌  |
+| snippets | id | No pagination | ✅ |  ❌  |
```

Comment:
I'd suggest to try manually checking if pagination works. If not most endpoints supports from-to parameters where you can get incremental + read all data in batches.

Reply:
@marcosmarxm I think incremental sync can be done as I've seen from and to parameters in some endpoints . will try that

Reply:
implemented constant backoff strategy of 5 secs for rate limiting and added pagination to some endpoints. @marcosmarxm